solidity/random-beacon/test/RandomBeaconGovernance.test.ts

```diff
@@ -18,6 +31,27 @@ describe("RandomBeaconGovernance", () => {
     randomBeacon = await RandomBeacon.deploy()
     await randomBeacon.deployed()
 
+    await randomBeacon.connect(governance).updateRelayEntryParameters(
+      initialRelayRequestFee,
+      initialRelayEntrySubmissionEligibilityDelay,
+      initialRelayEntryHardTimeout,
+      initialCallbackGasLimit
+    )
+    await randomBeacon.connect(governance).updateGroupCreationParameters(
+      initialGroupCreationFrequency,
+      initialGroupLifeTime,
+      initialDkgResultChallengePeriodLength,
+      initialDkgResultSubmissionEligibilityDelay
+    )
+    await randomBeacon.connect(governance).updateRewardParameters(
+      initialDkgResultSubmissionReward,
+      initialSortitionPoolUnlockingReward
+    )
+    await randomBeacon.updateSlashingParameters(
```

Comment:
`await randomBeacon.updateSlashingParameters` => `await randomBeacon.connect(governance).updateSlashingParameters`, just like in the above `update*` calls?